### DISCUSSION TITLE
v3.0.3

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1618] [Announcements]
## What's Changed
* fix(updater): fix rollback on stable releases by @mehalter in https://github.com/AstroNvim/AstroNvim/pull/1617

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.0.2...v3.0.3

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.0.3'>v3.0.3</a>.</em>

**Comment by kwiat1990** [comment]
I'm rather a beginner in subject of Nvim and setting up configuration with lazy but after deleting everything for v2 and installing v3 accordingly to steps from the official site: https://astronvim.com/#-setup I cannot get AstroNvim up and running without errors. 

I have copied and set up user config using this template: https://github.com/AstroNvim/user_example but after setting color scheme to `catpuccin` nothing has changed. Oddly enough after opening `~/.config/nvim` in AstroNvim I don't see `user` directory but if I `ls` direct in the terminal it's there. Additionally when I open one of existing config files, I get following error: 
<img width="1733" alt="image" src="https://user-images.githubusercontent.com/19494075/224563753-99183aeb-211d-4784-aae0-f880db69d12e.png">

The key binding for jumping between open buffers seems to be change or simply doesn't work anymore in v3.

**Reply by Uzaaft** [reply]
@kwiat1990 
First, for the issue with tree sitter, can you try: `TSInstall lua`
For the keybinding issue, please take a look at https://astronvim.com/nightly/Configuration/v3_migration

For the `user` dir issue, could you try and press `H`(capital h), and see if it appears?

**Reply by kwiat1990** [reply]
For some reason `user` dir was indeed hidden. It seems a bit odd. Installing lua helped also with the mentioned error inside config files. For color scheme to changed I needed to restart the terminal. Otherwise, I still saw the default one.
<img width="1733" alt="image" src="https://user-images.githubusercontent.com/19494075/224567830-84e5c348-3e89-44dc-aa3b-5a632ca8e3ad.png">
Btw. are those icons for `Session` or `Packages` correct or my font (`Hack Nerd Font Mono`) doesn't support those?

**Reply by luxus** [reply]
update your font
